shell/initialize/entry-helpers.js

```diff
@@ -264,12 +264,8 @@ export async function mountApp(appPartials, VueClass) {
 
   // Add beforeEach router hooks
   router.beforeEach(render.bind(vueApp));
-  router.beforeEach((from, to, next) => {
-    if (from?.name !== to?.name) {
```

Comment:
The check didn't seem to be necessary especially now that we won't be affected by redirects.